code/modules/mining/ores_coins.dm

```diff
@@ -17,6 +17,27 @@
 			user << "<span class='info'>Not enough fuel to smelt [src].</span>"
 	..()
 
+/obj/item/weapon/ore/Crossed(atom/movable/AM)
+	var/obj/item/weapon/storage/bag/ore/OB
+	var/turf/open/floor/F = get_turf(src)
+	if(loc != F)
+		return ..()
+	if(ishuman(AM))
+		var/mob/living/carbon/human/H = AM
+		for(var/thing in H.get_body_slots())

```

Comment:
why are we checking all the slots? are we expecting an ore bag in the glasses slot?
Also, we're not checking the hands themselves, the one place that would make sense.